environment/src/main/resources/application.yml

```diff
@@ -101,6 +101,16 @@ environment:
   enabledplatforms: AWS,AZURE,YARN,GCP
   enabledParentPlatforms: AWS,MOCK
   enabledChildPlatforms: YARN,MOCK
+  existing:
+    ssh:
+      key:
+        update:
+          support: AWS,MOCK
```

Comment:
ycloud ?

Reply:
what happens with if we add a new provider but that is not here

Reply:
We won't update the raw public key or public key id.